<?xml version="1.0"?>
<launch>


  <node pkg="ilcc" type="pcd_get_board" name="pcd_get_board" output="screen">
  <param name="box_z_min" value="-0.8"/>
  <param name="box_z_max" value="+1.5"/>
  <param name="box_x_min" value="-1.3"/>
  <param name="box_x_max" value="+0.5"/>
  <param name="box_y_min" value="-1.7"/>
  <param name="box_y_max" value="+1.5"/>
  <param name="save_board2pcd" value="false"/>
  <param name="pcd_save_path"  value="/home/ha/PCL_pcd/" />
  <param name="save_file_name"  value="boardCloud1" />

  </node>


</launch>

